Capgemini.Xrm.CdsDataMigrator/Capgemini.Xrm.CdsDataMigratorLibrary/Helpers/StaticPresenterHelpersWrapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace Capgemini.Xrm.CdsDataMigratorLibrary.Helpers
+{
+    public class StaticPresenterHelpersWrapper : IStaticPresenterHelpersWrapper
+    {
+        public bool AreAllCellsPopulated(DataGridViewRow row)
+        {
+            return PresenterHelpers.AreAllCellsPopulated(row);
+        }
+        public List<DataGridViewRow> GetMappingsFromViewWithEmptyRowsRemoved(List<DataGridViewRow> viewLookupMappings)
+        {
+            return PresenterHelpers.GetMappingsFromViewWithEmptyRowsRemoved(viewLookupMappings);
+        }
```

Comment:
I initially made the methods static since they don't modify the state of the object or use any instance variables. However based on my research it seems pretty tricky to mock static methods and so I put them in a wrapper in order to mock the wrapper. (Option 1 here: [http://adventuresdotnet.blogspot.com/2011/03/mocking-static-methods-for-unit-testing.html](url)

Do you think I'm best to scrap use of static methods and just make them instance methods?

Reply:
Yep, let's take out the static wrapper

Reply:
Removed static wrapper and mocking instance methods directly